cucumber/authenticators_jwt/features/authn_jwt_provider_uri.feature

```diff
@@ -0,0 +1,58 @@
+Feature: JWT Authenticator - JWKs fetched from Keycloak as OIDC provider
+
+  In this feature we define a JWT authenticator in policy and perform authentication
+  with Conjur. In successful scenarios we will also define a variable and permit the host to
+  execute it.
+
+  Background:
+    Given I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on Keycloak as OIDC provider.
+
+      - !variable
+        id: provider-uri
+
+      - !variable
+        id: token-app-property
+
+      - !variable
+        id: issuer
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: alice
+      annotations:
+        authn-jwt/keycloak/email: alice@conjur.net
+
+    - !grant
+      role: !group conjur/authn-jwt/keycloak/users
+      member: !host alice
+    """
+
+  Scenario: provider-uri has configured with valid value
+    Given I am the super-user
+    And I successfully set authn-jwt "provider-uri" variable with OIDC value from env var "PROVIDER_URI"
```

Comment:
I think we need to find a way that step in our jwt won't have the word OIDC

Reply:
I wrote OIDC because those environment variables are related to an OIDC provider